test/fixtures/cookbooks/test/recipes/default.rb

```diff
@@ -54,5 +54,12 @@
   action :import
 end
 
+# Importing the specified key
+gpg_key 'Import Ubuntu Key' do
+  keyserver 'keyserver.ubuntu.com'
+  key_fingerprint %w(409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB)
```

Comment:
When they change this it will break, so is there a way in test to get this value dynamically?